src/CMakeLists.txt

```diff
@@ -131,6 +131,10 @@ else()
         LIBS ${UMF_LIBS})
 endif()
 
+if(UMF_LINK_HWLOC_STATICALLY)
+    add_dependencies(umf hwloc)
```

Comment:
can't this be done right after the fetch (in the top-level CMakeLists)?

Reply:
`umf` target is created under src/, but fetch is performed level up, where the target doesn't exist.

Reply:
heh, make sense, I've mistaken umf lib with umf project